src/scripts/installer-queries/data-views.sql.fmt

```diff
@@ -111,9 +130,9 @@ CREATE OR REPLACE VIEW data.violation_suppression_rule_runs COPY GRANTS
   COMMENT='Stable interface to underlying metadata tables'
 AS
 SELECT V:RUN_ID::VARCHAR AS run_id
-  , V:QUERY_NAME AS query_name
-  , V:START_TIME AS start_time
-  , V:END_TIME AS end_time
+  , V:QUERY_NAME::VARCHAR AS query_name
+  , V:START_TIME::TIMESTAMP AS start_time
```

Comment:
Should the start_time and end_time be timestamps, or something like timestamp_tz?

Reply:
No strong feeling. I usually assume everything that isn't on a website in front of a lay-user is in UTC.